File: Modul 5/Modul 5 Bahasa C/PRAK502-2410817310007-MuhammadNaufalAbdillah.c
#include<stdio.h>
#include<math.h>
int hitungnilai (int nilai1, int nilai2){
    return sqrt((nilai2-nilai1) * (nilai2-nilai1));
}

int mutlak(int angka){
if (angka<0){
    return -angka;
}
return angka;
}

int main(){
    int a, b, c, d, hasil;
scanf("%d %d %d %d", &a, &c, &b, &d);

hasil= hitungnilai(a, b)+hitungnilai(c, d);

printf("%d", mutlak(hasil));
return 0;
}

File: Modul 5/Modul 5 Bahasa Python/PRAK502-2410817310007-MuhammadNaufalAbdillah.py
import math
def hitung_nilai(nilai1, nilai2):
    return math.sqrt((nilai2 - nilai1) ** 2)

def mutlak(angka):
    return abs(angka)

a, b, c, d = map(int, input().split())

hasil = hitung_nilai(a, c) + hitung_nilai(b, d)
print(mutlak(int (hasil)))